StreetLighting/StreetlightControlCabinet/doc/spec.md

```diff
@@ -75,7 +75,11 @@ responsible, district, neighbourhood, etc.
 + `dateLastProgramming` : Date at which there was a programming operation over the cabinet.
     + Attribute Type: [Date](http://schema.org/DateTime)
     + Optional
-    
+
++ `dateLastInvoice` : Date at which the last invoice for the cabinet was issued.
```

Comment:
we need to be more clear with the description

the last invoice corresponding to the energy consumed by the circuits controlled by this cabinet